src/Process/3dTilesProcessing.js

```diff
@@ -26,6 +27,27 @@ function subdivideNode(context, layer, node, cullingTest) {
     }
 }
 
+function boundingVolumeToExtent(crs, volume, transform) {
```

Comment:
What about having a `target` param as well, to avoid creating a new `Extent` if possible ?

Reply:
For now it's always used to create a `Extent` so a `target` param wouldn't be used.